src/cert_provider/main.cc

```diff
@@ -363,19 +384,23 @@ int main(int argc, char* argv[]) {
   boost::filesystem::path url_file = "gateway.url";
   if (!config_path.empty()) {
     Config config(config_path);
-    // TODO: provide path to root directory in `--local` parameter
+    // TODO(lbonn): provide path to root directory in `--local` parameter
```

Comment:
Why is lbonn's name getting injected in every TODO? I would've thought it would inject whoever last edited that line, but in this case, that would be Anton.